Dockerfile

```diff
@@ -18,161 +19,277 @@
+# ORT
+COPY . /usr/local/src/ort
 WORKDIR /usr/local/src/ort
 
-# Gradle build.
-RUN --mount=type=cache,target=/root/.gradle/ \
-    scripts/import_proxy_certs.sh && \
-    scripts/set_gradle_proxy.sh && \
-    sed -i -r 's,(^distributionUrl=)(.+)-all\.zip$,\1\2-bin.zip,' gradle/wrapper/gradle-wrapper.properties && \
-    ./gradlew --no-daemon --stacktrace -Pversion=$ORT_VERSION :cli:distTar :helper-cli:startScripts
-
-FROM adoptopenjdk:11-jre-hotspot-bionic
-
-ENV \
-    # Package manager versions.
-    BOWER_VERSION=1.8.8 \
-    BUNDLER_VERSION=1.16.1-1 \
-    CARGO_VERSION=0.47.0-1~exp1ubuntu1~18.04.1 \
-    COMPOSER_VERSION=1.6.3-1 \
-    CONAN_VERSION=1.18.0 \
-    GO_DEP_VERSION=0.5.4 \
-    GO_VERSION=1.16.5 \
-    HASKELL_STACK_VERSION=2.1.3 \
-    NPM_VERSION=6.14.2 \
-    PYTHON_PIPENV_VERSION=2018.11.26 \
-    PYTHON_VIRTUALENV_VERSION=15.1.0 \
-    SBT_VERSION=1.3.8 \
-    YARN_VERSION=1.22.4 \
-    # SDK versions.
-    ANDROID_SDK_VERSION=6858069 \
-    # Scanner versions.
-    SCANCODE_VERSION=3.2.1rc2 \
-    # Installation directories.
-    ANDROID_HOME=/opt/android-sdk \
-    GOPATH=$HOME/go
-
-ENV DEBIAN_FRONTEND=noninteractive \
-    PATH="$PATH:$HOME/.local/bin:$GOPATH/bin:/opt/go/bin:$GEM_PATH/bin"
-
-# Apt install commands.
-RUN --mount=type=cache,target=/var/cache/apt --mount=type=cache,target=/var/lib/apt \
-    apt-get update && \
-    apt-get install -y --no-install-recommends gnupg software-properties-common && \
-    echo "deb https://repo.scala-sbt.org/scalasbt/debian /" | tee -a /etc/apt/sources.list.d/sbt.list && \
-    curl -ksS "https://keyserver.ubuntu.com/pks/lookup?op=get&options=mr&search=0x2EE0EA64E40A89B84B2DF73499E82A75642AC823" | apt-key adv --import - && \
-    curl -sL https://deb.nodesource.com/setup_12.x | bash - && \
-    add-apt-repository -y ppa:git-core/ppa && \
-    apt-get update && \
-    apt-get install -y --no-install-recommends \
+#------------------------------------------------------------------------
+# This can be set to a directory containing CRT-files for custom certificates that ORT and all build tools should know about.
+ARG CRT_FILES=""
+COPY "$CRT_FILES" /tmp/certificates/
+ARG ORT_VERSION="DOCKER-SNAPSHOT"
+
+#------------------------------------------------------------------------
+# Gradle ORT build.
+RUN scripts/import_proxy_certs.sh \
+    && if [ -n "$CRT_FILES" ]; then \
+        /opt/ort/bin/import_certificates.sh /tmp/certificates/; \
+       fi \
+    && scripts/set_gradle_proxy.sh \
+    && . $NVM_DIR/nvm.sh \
+    && sed -i -r 's,(^distributionUrl=)(.+)-all\.zip$,\1\2-bin.zip,' gradle/wrapper/gradle-wrapper.properties \
+    && ./gradlew \
+        --no-daemon \
+        --stacktrace \
+        -Pversion=$ORT_VERSION \
+        -PscancodeVersion=${SCANCODE_VERSION} \
+        :cli:distTar \
+        :helper-cli:startScripts
+
+#------------------------------------------------------------------------
+# Remove the work directory
+RUN rm -rf /work
```

Comment:
It anything written to `/work`? I did not find anything when scrolling through

Reply:
Done, removed